pkg/client/add_events.go

```diff
@@ -417,3 +417,12 @@ func (client *DataSetClient) getBuffers() []*buffer.Buffer {
 	}
 	return buffers
 }
+
+// Truncate provided text to the provided length
```

Comment:
Yeah, I've truncated it for safety reasons in case API returns a really large response (unlikely). We could change the truncation logic or also increase the size at what we truncate since usually those non JSON responses won't be that large.

Reply:
>  It's really strange, that there is ni library function for this - https://pkg.go.dev/strings

Indeed. There are some upstream packages which do that it seems, so we could also try that instead of DIY.